src/components/dropdown-menu/DropdownMenuItemBase.tsx

```diff
@@ -0,0 +1,183 @@
+import * as radixDropdownMenu from '@radix-ui/react-dropdown-menu';
+import { MouseEventHandler, ReactNode } from 'react';
+import { IconCheck } from '../icons/IconCheck';
+
+type CheckProps =
+  | {
+      isCheckable: true;
+      isChecked?: boolean;
+    }
+  | {
+      isCheckable?: false;
+      isChecked?: never;
+    };
+
+export type DropdownMenuItemBaseProps = CheckProps & {
+  ariaLabel?: string;
+  children: ReactNode;
+  helpText?: JSX.Element | string;
+  isDisabled?: boolean;
+  isDestructive?: boolean;
+  onClick?: MouseEventHandler<HTMLDivElement>;
+  prefix?: JSX.Element;
+  suffix?: JSX.Element;
+};
+
+export function DropdownMenuItemBase({
+  ariaLabel,
+  children,
+  helpText,
+  isCheckable,
+  isChecked,
+  isDisabled,
+  isDestructive,
+  onClick,
+  prefix,
+  suffix,
+}: Readonly<DropdownMenuItemBaseProps>) {
+  let checkMarkOrPlaceHolder = null;
+
+  if (isCheckable) {
+    checkMarkOrPlaceHolder = isChecked ? (
+      <IconCheck color="echoes-color-icon-selected" />
+    ) : (
+      <CheckmarkPlaceholder />
+    );
+  }
+
+  return (
+    <StyledDropdownMenuItemBase
+      disabled={isDisabled}
+      isCheckable={isCheckable}
+      isChecked={isChecked}
+      isDestructive={isDestructive}
+      onClick={isDisabled ? undefined : onClick}>
+      <StyledLeftHandSide>
+        {checkMarkOrPlaceHolder}
+
+        {prefix}
+
+        <StyledLabelAndHelpText aria-label={ariaLabel}>
+          {children}
+
+          <StyledHelpText isDisabled={isDisabled}>{helpText}</StyledHelpText>
+        </StyledLabelAndHelpText>
+      </StyledLeftHandSide>
+      <StyledSuffix>{suffix}</StyledSuffix>
+    </StyledDropdownMenuItemBase>
+  );
+}
+
+const CheckmarkPlaceholder = styled(IconCheck)`
+  visibility: hidden;
+`;
+
+const StyledLeftHandSide = styled.div`
+  align-items: center;
+  column-gap: var(--echoes-dimension-space-100);
+  display: flex;
+`;
+
+const StyledLabelAndHelpText = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  row-gap: var(---echoes-dimension-space-25);
+`;
+
+const StyledHelpText = styled.span<{ isDisabled?: boolean }>`
+  color: ${(props) =>
+    props.isDisabled ? 'var(--echoes-color-text-disabled)' : 'var(--echoes-color-text-subdued)'};
+  font-size: var(--echoes-font-size-body-small);
+`;
+
+const StyledSuffix = styled.span`
+  align-items: center;
+  color: var(--echoes-color-text-subdued);
+  display: flex;
+`;
+
+const StyledDropdownMenuItemBase = styled(radixDropdownMenu.Item)<
+  Pick<DropdownMenuItemBaseProps, 'isCheckable' | 'isChecked' | 'isDestructive'>
+>`
+  align-items: center;
+  background-color: ${(props) =>
+    props.isChecked
+      ? 'var(--echoes-color-background-selected-weak-default)'
+      : 'var(--echoes-color-background-default)'};
+  color: ${(props) =>
+    props.isDestructive ? 'var(--echoes-color-text-danger)' : 'var(--echoes-color-text-default)'};
+  column-gap: var(--echoes-dimension-space-100);
+  cursor: pointer;
```

Comment:
I don't think we should force a `cursor: pointer`

Reply:
why not? All items in a dropdown are supposed to be clickable if they're not disabled (or labels, separators,..)
If we don't add it here then every "customer" will have to add it themselves

Reply:
but this is what we do everywhere - what other cursor should be used? Some items will be links, some buttons - should we have different cursors?

Reply:
let's keep it for now, as discussed. @marciopmoreira6 thinks we should keep it to, JIRA does the same thing :shrug: